### DISCUSSION TITLE
Livecheck every installed formula?

### DISCUSSION THREAD

**Discussion by viking1304** [Homebrew/discussions#2853] [Everyday usage] [answered]
Is there an easy way to do livecheck every installed formula? Something similar like `brew outdated`, but that would actually do live check for every formula.

I want to be able to easily check if any formula that I am using is outdated, so I can make bump PR.

I can easily make a simple bash script that will loop trough output of `brew list` and do the checks, but I am trying to figure out if there is a more elegant way, since brew has many nicely done checks out of the box.

**Comment by SMillerDev** [comment, marked as answer]
Like `brew livecheck --installed`?

**Reply by viking1304** [reply]
> Like `brew livecheck --installed`?

Close enough!

Actually this was what I was looking for:

`
❯ brew livecheck --installed --newer-only
`

As expected, Homebrew always have some nice and elegant solution for almost everything I can ever think of. :)

**Reply by SMillerDev** [reply]
We even have `brew livecheck --help` to help you figure out which parameters/flags to pick. 😄

**Reply by viking1304** [reply]
> We even have `brew livecheck --help` to help you figure out which parameters/flags to pick. 😄

Absolutely! `--help` should be everyones best friend. :)
